google-cloud-firestore/src/main/java/com/google/cloud/firestore/AggregateQuery.java

```diff
@@ -340,15 +340,15 @@ public void onError(Throwable throwable) {
             .currentSpan()
             .addEvent(
                 SPAN_NAME_RUN_AGGREGATION_QUERY + ": Retryable Error",
-                Collections.singletonMap("error.message", throwable.getMessage()));
+                Collections.singletonMap("error.message", throwable.toString()));
```

Comment:
what is the motivation for using toString instead of getMessage? should other files be changed as well?

Reply:
this is the implementation of `toString()`:

```java
    public String toString() {
        String s = getClass().getName();
        String message = getMessage();
        return (message != null) ? (s + ": " + message) : s;
    }
```

Using toString() is safer than getMessage because toString handles cases where message is null. 

This PR has fixed it for all cases related to tracing. I think there are a couple of other instances that we should fix too [in a separate PR].